### PR TITLE
Doc 9

### PR DESCRIPTION
**Pull request by janettelynch**

### CODE REVIEW

File: guides/compliance/compliance-controls.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+![](../../assets/compliance-controls-overview.png)
+
+Click a control to add evidence, see how many compliance requirements currently link to it, and any 
+[linked policies and procedures](./compliance-mapping-policies.md).
+
+![](../../assets/compliance-control-detail.png)
+
+
+
+When you link a control, you are given the option of using all the current and future evidence items from the linked control for the gap evaluation of this compliance item. If you select this option, J1 automatically monitors the control in the context of the linked evidence for the requirement.
```
Comment (henrymgarrett) [inline guides/compliance/compliance-controls.md:20]: Nit: "When you link a control to a framework"

File: guides/compliance/compliance-import.md [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+**Note:** You must have the adminCompliance permission to be able to import frameworks.
 
-[plus-icon]: https://raw.githubusercontent.com/feathericons/feather/master/icons/plus-circle.svg?sanitize=true
+Open the J1 Compliance app by clicking ![](../../assets/icons/apps.png)and select Compliance.  To add a framework:
 
-See this github repository for example compliance standards in JSON format:
+1. Select **Frameworks** from the left navigation panel and click **Add Framework**.
+   ​
+   ![](../../assets/compliance-frmwk-overview.png)
+2. Choose one of the templates that J1 provides, or click **Import JSON/CVS** and
```
Comment (henrymgarrett) [inline guides/compliance/compliance-import.md:20]:
```suggestion
2. Choose one of the templates that J1 provides, or click **Import JSON/CSV** and 
```

File: guides/compliance/compliance-mapping-policies.md [hunk 3]
```diff
@@ -13,49 +13,14 @@ security_procedure
...
-> Mapping is created only between security procedures to a compliance
-requirement/control. Policies are indirectly mapped via procedures. This is
-because policies are very high-level descriptions (such as principal statements)
-and principals cannot directly implement a compliance requirement/control. Only
-details on what you do and how you do it (the procedures) can implement a
-requirement/control.
->
-> ![compliance-mapped-policy-procedure](../../assets/compliance-mapped-policy-procedure.png)
+When you link a policy and procedure, J1 creates a link between a security procedure to a 
+compliance requirement or control. Policies are very high-level descriptions, that provide
```
Comment (henrymgarrett) [inline guides/compliance/compliance-mapping-policies.md:17]: It is just to a compliance control, we dont link to compliance requirements anymore

File: guides/compliance/compliance-mapping-policies.md [hunk 4]
```diff
@@ -13,49 +13,14 @@ security_procedure
...
+When you link a policy and procedure, J1 creates a link between a security procedure to a 
+compliance requirement or control. Policies are very high-level descriptions, that provide 
+details on what you should do. Procedures explain how you can implement a requirement 
+or control.
 
-## Edit Mapping of an Individual Compliance Requirement or Control
-
-Mapped security procedures of a specific compliance requirement or control can
-be selected via the UI in the requirement details view:
+You can link security procedures for a specific compliance requirement or control in the details view
```
Comment (henrymgarrett) [inline guides/compliance/compliance-mapping-policies.md:21]: Same here, can only link to a control

**Review by AnneNielsenJ1 — COMMENTED** [review]
Looks good! I have no edits but would like @sarabeth to give the final sign off. Thank you Janette!

**Review by sarabeth — APPROVED** [review]

**Pull request merged**